src/java/org/jivesoftware/openfire/muc/spi/IQMUCSearchHandler.java

```diff
@@ -151,41 +155,41 @@ public IQ handleIQ(IQ iq)
             }
         }
 
-        final FormField matchFF = df.getField("name_is_exact_match");
+        final FormField matchFF = df.getField("nameIsExactMatch");
```

Comment:
Did you really mean to change the name of the field, `"name_is_exact_match"` -> `"nameIsExactMatch"` or did you actually mean to introduce a constant `nameIsExactMatch` ?

Reply:
Hi Greg. This is just a Boolean variable declared on line 143. I just renamed it to the camel case standard instead of separating w/underscores. So yes I just meant to change the name of the field.